src/pages/index.js

```diff
@@ -153,9 +288,21 @@ function closeModal(modal) {
 
 function saveProfileChanges(evt) {
   evt.preventDefault();
-  profileName.textContent = modalName.value;
-  profileDescription.textContent = modalDescription.value;
-  closeModal(editProfileModal);
+
+  const submitBtn = evt.submitter;
+  submitBtn.textContent = "Saving...";
```

Comment:
Please, reuse ` renderLoading(true, submitBtn);` when you change the profile data as well not to duplicate the code